framework/src/main/java/org/checkerframework/framework/util/DefaultAnnotationFormatter.java

```diff
@@ -28,8 +28,8 @@ public static boolean isInvisibleQualified(AnnotationMirror anno) {
     }
 
     /**
-     * Creates a space String of each annotation in annos separated by a single space character,
-     * obeying the printInvisible parameter.
+     * Creates a space String of each annotation in annos separated by a single space character and
```

Comment:
Can the first `space` be removed? I'm not sure what a `space String` would be.

Reply:
Yes, that's a pre-existing typo.  I'll fix it.